web/src/js/components/Dashboard/VideoEngagementComponent.js

```diff
@@ -79,30 +79,86 @@ const VideoEngagementComponent = ({
       closeModal()
     }
   }, [status])
-
   return (
     <div>
-      <ButtonBase onClick={openAd} disabled={!adAvailable}>
-        <MovieFilterIcon
-          {...iconProps}
+      <div
+        ref={poppoverRef}
+        onClick={!adAvailable ? () => setIsPoppoverOpen(true) : undefined}
+      >
+        <ButtonBase
+          onClick={openAd}
+          disabled={!adAvailable}
+          onMouseEnter={adAvailable ? () => setIsPoppoverOpen(true) : undefined}
+        >
+          <MovieFilterIcon
+            {...iconProps}
+            style={{
+              color: adAvailable ? 'gold' : disabledColor,
+            }}
+          />
+        </ButtonBase>
+      </div>
+      <DashboardPopover
+        anchorEl={poppoverRef.current}
+        onClose={
+          adAvailable
+            ? () => {
+                setIsPoppoverOpen(false)
+                openAd()
```

Comment:
@kmjennison this behavior made sense to me.  The icon shows gold, they hover and read the text, then they click the icon again to trigger this onClose which opens the ad.  Alternative is that they click once to close and click again to open ad.  You can test behavior on dev